scripts/generate_config_yaml.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python3
```

Comment:
The vast majority of other python scripts in ROS 2 use `#!/usr/bin/env python3` (47 instances vs 3).
```suggestion
#!/usr/bin/env python3
```